### ISSUE TITLE
more informative error for flywire_xyz2id when coordinate space might be wrong

### ISSUE THREAD

**Issue by alexanderbates** [natverse/fafbseg#78]
This works:

```
# get ID
flywire.xyz = "108847.975,62454.2775,3811"
flywire.id = fafbseg::flywire_xyz2id(nat::xyzmatrix(flywire.xyz), rawcoords = TRUE)
```

However, earlier today I forgot that the default was not rawcoords = TRUE. If I do it with FALSE, I get this error:

```
flywire.xyz = "108847.975,62454.2775,3811"
flywire.id = fafbseg::flywire_xyz2id(nat::xyzmatrix(flywire.xyz), rawcoords = FALSE)
> Error in parse_url(url) : length(url) == 1 is not TRUE
```

And didn't immediately realise what was wrong. I also happened to get two separate inquiries over the last few days about why this function was not working, both times it was the same issue.

More informative error message needed?

**Comment by alexanderbates** [comment]
@tatsuookubo this was the issue we had ealier

**Comment by jefferis** [comment]
closed by #82
